style.css

```diff
@@ -0,0 +1,136 @@
+*{
+    margin:0;
+    padding:0;
+}
+header img{
+    height: 70px;
+    margin-left: 10%;
```

Comment:
This is not the correct way to space elements.

If you want a gutter space, you should drive that through the parent element. So here, drive this space by giving padding to the header element